static/js/whyis_vue/components/table-view/frontend/src/App.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div id="app">
+    <MainPage
+      @apiPost="testApiPost"
+      @apiGet="testApiGet"
+    />
+  </div>
+</template>
+
+<script>
+/* eslint-disable */
+import MainPage from './components/HelloWorld.vue';
```

Comment:
File name should be the same as the component name.

Reply:
Fixed